src/store/reducers/apiCalls.ts

```diff
@@ -42,3 +42,14 @@ export const reducer: Reducer<ApiCallsState> = (
     endpointCalls.set(action.identifier, { status })
   );
 };
+
+function getRandomInt(max: number) {
+  return Math.floor(Math.random() * Math.floor(max));
+}
+
+/**
+ * Returns a randomly generated identifier for an API call.
+ */
+export function generateRandomIdentifier() {
```

Comment:
Just to be doubly-sure we never use this instead of something like https://caniuse.com/#feat=getrandomvalues, let's add comment saying not safe for crypto.